### PR TITLE
Add RubiconProject account to tests

### PR DESCRIPTION
**Pull request by yoalex5**
GitHub issue #174

### CODE REVIEW

**Comment by yoalex5** [comment]
@ppuviarasu could you please review this PR?

Thank you

File: example/Swift/PrebidDemo/PrebidDemoTests/PBViewTool.swift [hunk 1]
```diff
@@ -87,17 +87,21 @@ class PBViewTool: NSObject {
...
             let name: String = String(describing: type(of: view))
-            if (name == "GADNWebAdView" || name == "GADOAdView") {
+            if (name == "GADNWebAdView" || name == "GADOAdView" || name == "GADWebAdView") {
                 let views  = view.subviews
                 for innerView in views {
                     let nameInner: String = String(describing: type(of: innerView))
                     if (innerView.isKind(of: WKWebView.self)) {
-                      return PBViewTool.checkJSExistInWebView(webView: innerView as! WKWebView)
+                      return PBViewTool.checkJSExistInWebView(wkWebView: innerView as! WKWebView)
+                    } else if (innerView.isKind(of: UIWebView.self)) {
```
Comment (ppuviarasu) [inline example/Swift/PrebidDemo/PrebidDemoTests/PBViewTool.swift]: We are supporting iOS 10 & above... Why are we checking for UIWebView?

Comment (yoalex5) [reply example/Swift/PrebidDemo/PrebidDemoTests/PBViewTool.swift]: It seems that AdManager uses UIWebView

Screen Shot 2019-04-18 at 7 07 46 PM

iPhone 7 
iOS 12.2
Google-Mobile-Ads-SDK v7.34.0

Comment (ppuviarasu) [reply example/Swift/PrebidDemo/PrebidDemoTests/PBViewTool.swift]: sounds good

**Pull request merged**